<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ajoutTransporteur</class>
 <widget class="QDialog" name="ajoutTransporteur">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Transporteur</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Nom</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nomEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Prenom</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="prenomEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Telephone</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="telEdit">
       <property name="inputMask">
        <string/>
       </property>
       <property name="maxLength">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Adresse</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPlainTextEdit" name="adresseEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btVlider">
       <property name="text">
        <string>Valider</string>
       </property>
       <property name="icon">
        <iconset resource="../Res.qrc">
         <normaloff>:/icones/Icones/15-Tick-icon.png</normaloff>:/icones/Icones/15-Tick-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btAnnuler">
       <property name="text">
        <string>Annuler</string>
       </property>
       <property name="icon">
        <iconset resource="../Res.qrc">
         <normaloff>:/icones/Icones/19-Left-Arrow-icon.png</normaloff>:/icones/Icones/19-Left-Arrow-icon.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btFermer">
       <property name="text">
        <string>Fermer</string>
       </property>
       <property name="icon">
        <iconset resource="../Res.qrc">
         <normaloff>:/icones/Icones/gnome_application_exit.png</normaloff>:/icones/Icones/gnome_application_exit.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../Res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btAnnuler</sender>
   <signal>clicked()</signal>
   <receiver>telEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>315</x>
     <y>71</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAnnuler</sender>
   <signal>clicked()</signal>
   <receiver>prenomEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAnnuler</sender>
   <signal>clicked()</signal>
   <receiver>nomEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>349</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>12</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btFermer</sender>
   <signal>clicked()</signal>
   <receiver>ajoutTransporteur</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>430</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>139</x>
     <y>115</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btAnnuler</sender>
   <signal>clicked()</signal>
   <receiver>adresseEdit</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>170</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>145</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
